pkg/solana/types.go

```diff
@@ -27,11 +27,13 @@ const (
 // State is the struct representing the contract state
 type State struct {
 	AccountDiscriminator [8]byte // first 8 bytes of the SHA256 of the account’s Rust ident, https://docs.rs/anchor-lang/0.18.2/anchor_lang/attr.account.html
+	Version              uint8
 	Nonce                uint8
+	Padding0             uint16
+	Padding1             uint32
```

Comment:
Consider something like this, as the paddings are unused nd just reserve memory:
```suggestion
	_ /* _padding0 */    uint16
	_ /* _padding0 */    uint32
```